hvplot/interactive.py

```diff
@@ -169,7 +169,16 @@ def _clone(self, transform=None, plot=None, loc=None, center=None,
             kwargs = dict(self._kwargs, inherit_kwargs=self._inherit_kwargs, method=self._method, **kwargs)
         else:
             kwargs = dict(self._inherit_kwargs, **dict(self._kwargs, **kwargs))
-        return type(self)(self._obj, fn=self._fn, transform=transform, plot=plot, depth=depth,
+
+        # This is a hack to avoid calling the original bound function multiple times
+        # when cloning. self._obj is updated by the orignal function with depth=0
+        if self._fn:
+            f = pn.panel(pn.bind(lambda *args, **kwargs: self._obj), lazy=True)
+            f.object._dinfo = self._fn.object._dinfo
```

Comment:
Very clever but also additional layers of indirection. Let's go with https://github.com/holoviz/hvplot/pull/818

Reply:
I completely agree that #818 is the better choice.